ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ebgp/peer-groups.j2

```diff
@@ -12,3 +12,15 @@
 {% if switch.evpn_role == "server" %}
       next_hop_unchanged: true
 {% endif %}
+{% if switch.evpn_gateway is arista.avd.defined(true) %}
```

Comment:
Same concern here as above, just checking for switch.evpn_gateway doesn't take into account future need for multiple gateway peer groups on the same box.